app/src/main/res/values/strings.xml

```diff
@@ -93,48 +93,48 @@
     <string name="app_setting">App settings</string>
     <string name="column_list">Columns list</string>
     <string name="update_access_token">Update the access token</string>
-    <string name="account_remove">Remove this account</string>
+    <string name="account_remove">Remove account</string>
     <string name="actions">Actions</string>
-    <string name="default_status_visibility">Default visibility of toot</string>
-    <string name="confirm_before_boost">Confirm before boost</string>
+    <string name="default_status_visibility">Default toot visibility</string>
+    <string name="confirm_before_boost">Confirm before boosting</string>
     <string name="sensitive_content_default_open">Always open sensitive content</string>
     <string name="user">User</string>
-    <string name="confirm_account_remove">This account will be deleted and all the columns will be removed.\nAre you sure?</string>
+    <string name="confirm_account_remove">Remove account along with all columns?</string>
     <string name="user_name_not_match">Username doesn\'t match</string>
     <string name="access_token_updated_for">%1$s\'s access token was updated.</string>
-    <string name="back_to_column_list">Go to columns list when back button is pressed</string>
-    <string name="account_empty">Account is not set up.Please add account.</string>
+    <string name="back_to_column_list">Go to columns list when pressing \"←\"</string>

```

Comment:
looks of back button can be displayed as  ←, < , and rotated △. 
also we can find  ← in IME's software keyboard for backspace and left cursor key.
I oppose to mimic glyphs of a particular design.

Reply:
Well, I don't think it does on Android. If so, how does Android render the symbol natively?